ui/src/theme/headerStyles.js

```diff
@@ -13,7 +13,7 @@ const {
 
 export const HEADER_HEIGHT = '64px';
 
-export const Header = styled('header')`
+export const HeaderWrapper = styled('div')`
```

Comment:
@mistryrn for semantics
```suggestion
export const HeaderWrapper = styled('section')`
```